File: src/test/java/no/nav/syfo/service/MoterUtilTest.java
package no.nav.syfo.service;

import no.nav.syfo.domain.model.MotedeltakerAktorId;
import no.nav.syfo.util.MoterUtil;
import org.junit.Test;
import org.junit.runner.RunWith;
import org.mockito.runners.MockitoJUnitRunner;

import java.util.Arrays;

import static java.time.LocalDateTime.now;
import static org.assertj.core.api.Assertions.assertThat;

@RunWith(MockitoJUnitRunner.class)
public class MoterUtilTest {

    @Test
    public void sisteSvarErMottatt() {
        boolean erSisteSvarMottatt = MoterUtil.erSisteSvarMottatt(Arrays.asList(
                new MotedeltakerAktorId().svartTidspunkt(now())
        ));

        assertThat(erSisteSvarMottatt).isTrue();
    }

    @Test
    public void sisteSvarErIkkeMottatt() {
        boolean erSisteSvarMottatt = MoterUtil.erSisteSvarMottatt(Arrays.asList(
                new MotedeltakerAktorId().svartTidspunkt(null)
        ));

        assertThat(erSisteSvarMottatt).isFalse();
    }
}

File: src/test/kotlin/no/nav/syfo/service/MoterUtilTest.kt
package no.nav.syfo.service

import no.nav.syfo.domain.model.MotedeltakerAktorId
import no.nav.syfo.util.MoterUtil
import org.assertj.core.api.Assertions
import org.junit.Test
import org.junit.runner.RunWith
import org.mockito.junit.MockitoJUnitRunner
import java.time.LocalDateTime

@RunWith(MockitoJUnitRunner::class)
class MoterUtilTest {
    @Test
    fun sisteSvarErMottatt() {
        val erSisteSvarMottatt = MoterUtil.erSisteSvarMottatt(
            listOf(
                MotedeltakerAktorId().svartTidspunkt(LocalDateTime.now())
            )
        )
        Assertions.assertThat(erSisteSvarMottatt).isTrue
    }

    @Test
    fun sisteSvarErIkkeMottatt() {
        val erSisteSvarMottatt = MoterUtil.erSisteSvarMottatt(
            listOf(
                MotedeltakerAktorId().svartTidspunkt(null)
            )
        )
        Assertions.assertThat(erSisteSvarMottatt).isFalse
    }
}
